sdcard/rootfs/root/pwnagotchi/scripts/pwnagotchi/ui/view.py

```diff
@@ -43,13 +43,22 @@ def setup_display_specifics(config):
 
     elif config['ui']['display']['type'] in ('ws_1', 'ws1', 'waveshare_1', 'waveshare1',
                                              'ws_2', 'ws2', 'waveshare_2', 'waveshare2'):
-        fonts.setup(10, 9, 10, 35)
+        if config['ui']['display']['color'] == 'black':
+            fonts.setup(10, 9, 10, 35)
+
+            width = 250
+            height = 122
+            face_pos = (0, 40)
+            name_pos = (125, 20)
+            status_pos = (125, 35)
+        else:
+            fonts.setup(10, 8, 10, 25)
 
-        width = 250
```

Comment:
these are default values and should not be changed!

Reply:
I'll test to see if it works with the regular dimensions, but I'm pretty sure the waveshare v1 b/c 3-color models are smaller in terms of pixel dimensions. Regarding font changes, I'll go ahead and test those out; they were there from the code I forked off of.

Reply:
@evilsocket the value changes are the same as the stuff for Inky. Waveshare 3-color has the same pixel sizes as Inky, so Waveshare 3-color should be the same configuration as Inky in this case

References:
https://shop.pimoroni.com/products/inky-phat
https://www.waveshare.com/product/modules/oleds-lcds/e-paper/2.13inch-e-paper-hat-b.htm

Reply:
yeah but not the other waveshares